specs/_features/eip7594/das-core.md

```diff
@@ -55,6 +55,7 @@ We define the following Python custom types for type hinting and readability:
 | Name | Value | Description |
 | - | - | - |
 | `NUMBER_OF_COLUMNS` | `uint64(FIELD_ELEMENTS_PER_EXT_BLOB // FIELD_ELEMENTS_PER_CELL)` (= 128) | Number of columns in the extended data matrix. |
+| `MAX_EXTENDED_MATRIX_SIZE` | `uint64(MAX_BLOBS_PER_BLOCK * NUMBER_OF_COLUMNS)` (= 768) | The data size of `ExtendedMatrix`. |
```

Comment:
@kevaundray 

Re: the `error: Invalid base class "List"` issue

Pylint can't interpret `class ContainerName(List[c, a*b])` correctly. 

I added a workaround by adding a new config, `MAX_EXTENDED_MATRIX_SIZE`, to replace `AX_BLOBS_PER_BLOCK * NUMBER_OF_COLUMNS`.

Another workaround is to treat it as a special case by adding a `# type: ignore` by updating
https://github.com/ethereum/consensus-specs/blob/9c04cf13c95a355b95b09d605fd10185b0995b41/pysetup/helpers.py#L25-L29

I prefer the former a bit.

Reply:
I have no preference, so whatever you think is best :)